devservices/config/relay.yml

```diff
@@ -0,0 +1,21 @@
+---
+relay:
+  upstream: "http://host.docker.internal:8001/"
+  host: 0.0.0.0
+  port: 7899
+logging:
+  level: INFO
+  enable_backtraces: false
+limits:
+  shutdown_timeout: 0
+processing:
+  enabled: true
+  kafka_config:
+    - { name: "bootstrap.servers", value: "kafka:9093" }
+    # The maximum attachment chunk size is 1MB. Together with some meta data,
+    # messages will never get larger than 2MB in total.
+    - { name: "message.max.bytes", value: 2097176 }
+  topics:
+    metrics_transactions: ingest-performance-metrics
+    metrics_sessions: ingest-metrics
```

Comment:
```suggestion
```

`metrics_transactions` doesn't exist. `metrics_sessions` already defaults to `ingest-metrics`

See here: https://github.com/getsentry/relay/blob/c63be1331a75091bef0e046c1090d288a53495e9/relay-kafka/src/config.rs#L123-L138

Reply:
Got it, I got this config from https://github.com/getsentry/sentry/tree/master/config/relay. Sounds like more of a reason to have all relay configs live in the relay repo if possible